packages/fluent-bit-lsp/package.yaml

```
@@ -0,0 +1,42 @@
+---
+name: fluent-bit-lsp
+description: Language server for fluent-bit config file.
+homepage: https://github.com/sh-cho/fluent-bit-lsp
+licenses:
+  - Apache-2.0
+  - MIT
+languages:
+  - fluentbit
```

```suggestion
  - Fluent Bit
```

I believe this is the correct casing and spacing from their documentation. This isn't the same as `filetype` in vim